Duplicati/Library/Backend/OpenStack/OpenStackStorage.cs

```diff
@@ -68,6 +68,7 @@ public class OpenStackStorage : IBackend, IStreamingBackend
             new KeyValuePair<string, string>("OVH Cloud Storage", "https://auth.cloud.ovh.net/v3"),
             new KeyValuePair<string, string>("Selectel Cloud Storage", "https://auth.selcdn.ru"),
             new KeyValuePair<string, string>("Memset Cloud Storage", "https://auth.storage.memset.com"),
+            new KeyValuePair<string, string>("Infomaniak SwissBackup", "https://swiss-backup02.infomaniak.com/identity/v3"),

```

Comment:
Thanks @PAPAMICA.  Would `Infomaniak Swiss Backup` (with a space between `Swiss` and `Backup`) be more consistent with the [product name](https://www.infomaniak.com/en/swiss-backup)?